app/src/test/java/rick_and_morty/ui/episodes/EpisodesViewModelTest.kt

```diff
@@ -30,8 +34,19 @@ class EpisodesViewModelTest {
     )
 
     private val episodesRepository: EpisodesRepository = mock()
+    private val realm: Realm = mock() // Create a mock of Realm
 
-    private val classToTest by lazy { EpisodesViewModel(episodesRepository) }
+    // Define the Realm results mock
+    private val realmResults: RealmResults<RealmEpisodes> = mock {
+        on { map { it.toEpisodesResultDto() } } doReturn mockEpisodes
+    }
+
+    // Mock the realm queries
+    init {
+        given(realm.where(RealmEpisodes::class.java).findAll()).willReturn(realmResults)
+    }
+
```

Comment:
This is usually done in a `@Setup`-annotated method in unit tests. This way it will be triggered before each test.

Reply:
Done